### PR TITLE
feat:Add single test to ByteTimeUtils and fix code that triggers null…

### PR DESCRIPTION
**Pull request by ahaoyao**
feat:Add single test to ByteTimeUtils and fix code that triggers null pointer exception

### CODE REVIEW

File: linkis-commons/linkis-common/src/test/java/org/apache/linkis/common/utils/ByteTimeUtilsTest.java [hunk 1]
```diff
@@ -17,35 +17,156 @@
...
+    convertToGB.forEach(
+        (k, v) -> Assertions.assertEquals(opFunction.get("byteStringAsGb").apply(k), v));
+    Assertions.assertThrows(
+        IllegalArgumentException.class, () -> opFunction.get("byteStringAsGb").apply("1C"));
+  }
 
-    }
-}
\ No newline at end of file
+  private static final ImmutableMap<String, Long> convertToByte =
```
Comment (casionone) [inline linkis-commons/linkis-common/src/test/java/org/apache/linkis/common/utils/ByteTimeUtilsTest.java:68]: about static code 
best to put it in front of the test

Comment (ahaoyao) [reply linkis-commons/linkis-common/src/test/java/org/apache/linkis/common/utils/ByteTimeUtilsTest.java:36]: done
